2013/10/22 22:55:10 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in C:\xampp\htdocs\FACIS\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\xampp\htdocs\FACIS\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\FACIS\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\FACIS\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\FACIS\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\FACIS\framework\gii\generators\model\ModelCode.php(57): CModule->__get('db')
#5 C:\xampp\htdocs\FACIS\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\FACIS\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\FACIS\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\FACIS\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\FACIS\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\FACIS\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\FACIS\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\FACIS\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\FACIS\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 C:\xampp\htdocs\FACIS\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\FACIS\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/facis/index.php/gii/model
HTTP_REFERER=http://localhost:8081/facis/index.php/gii
---
2013/10/22 23:09:27 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1400 OCIStmtExecute: ORA-01400: cannot insert NULL into ("FACIS"."SOCIO"."K_IDENTIFICACION")
 (ext\pdo_oci\oci_statement.c:148). The SQL statement executed was: INSERT INTO "SOCIO" ("N_NOMBRE", "N_APELLIDO", "I_ESTADO_CIVIL", "N_OCUPACION", "O_TARJETA_PROFESIONAL", "I_GENERO", "O_DIRECCION_DOMICILIO", "O_DIRECCION_TRABAJO", "O_CORREO_ELECTRONICO", "O_TELEFONO_DOMICILIO", "O_TELEFONO_TRABAJO", "O_TELEFONO_CELULAR", "F_INGRESO", "F_RETIRO", "O_CAUSAL_RETIRO") VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14) RETURNING "K_IDENTIFICACION" INTO :RETURN_ID.
in C:\xampp\htdocs\FACIS\protected\controllers\SOCIOController.php (73)
in C:\xampp\htdocs\FACIS\index.php (13)
2013/10/22 23:09:27 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1400 OCIStmtExecute: ORA-01400: cannot insert NULL into ("FACIS"."SOCIO"."K_IDENTIFICACION")
 (ext\pdo_oci\oci_statement.c:148). The SQL statement executed was: INSERT INTO "SOCIO" ("N_NOMBRE", "N_APELLIDO", "I_ESTADO_CIVIL", "N_OCUPACION", "O_TARJETA_PROFESIONAL", "I_GENERO", "O_DIRECCION_DOMICILIO", "O_DIRECCION_TRABAJO", "O_CORREO_ELECTRONICO", "O_TELEFONO_DOMICILIO", "O_TELEFONO_TRABAJO", "O_TELEFONO_CELULAR", "F_INGRESO", "F_RETIRO", "O_CAUSAL_RETIRO") VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14) RETURNING "K_IDENTIFICACION" INTO :RETURN_ID' in C:\xampp\htdocs\FACIS\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\xampp\htdocs\FACIS\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\xampp\htdocs\FACIS\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\FACIS\protected\controllers\SOCIOController.php(73): CActiveRecord->save()
#3 C:\xampp\htdocs\FACIS\framework\web\actions\CInlineAction.php(49): SOCIOController->actionCreate()
#4 C:\xampp\htdocs\FACIS\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\FACIS\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\FACIS\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\FACIS\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\FACIS\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\FACIS\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\FACIS\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\FACIS\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\FACIS\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\htdocs\FACIS\framework\web\CWebApplication.php(141): CWebApplication->runController('sOCIO/create')
#14 C:\xampp\htdocs\FACIS\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\FACIS\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/facis/index.php/sOCIO/create
HTTP_REFERER=http://localhost:8081/facis/index.php/sOCIO/create
---
2013/10/22 23:12:01 [warning] [application] Failed to set unsafe attribute "K_IDENTIFICACION" of "SOCIO".
in C:\xampp\htdocs\FACIS\protected\controllers\SOCIOController.php (72)
in C:\xampp\htdocs\FACIS\index.php (13)
2013/10/22 23:12:01 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1400 OCIStmtExecute: ORA-01400: cannot insert NULL into ("FACIS"."SOCIO"."K_IDENTIFICACION")
 (ext\pdo_oci\oci_statement.c:148). The SQL statement executed was: INSERT INTO "SOCIO" ("N_NOMBRE", "N_APELLIDO", "I_ESTADO_CIVIL", "N_OCUPACION", "O_TARJETA_PROFESIONAL", "I_GENERO", "O_DIRECCION_DOMICILIO", "O_DIRECCION_TRABAJO", "O_CORREO_ELECTRONICO", "O_TELEFONO_DOMICILIO", "O_TELEFONO_TRABAJO", "O_TELEFONO_CELULAR", "F_INGRESO", "F_RETIRO", "O_CAUSAL_RETIRO") VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14) RETURNING "K_IDENTIFICACION" INTO :RETURN_ID.
in C:\xampp\htdocs\FACIS\protected\controllers\SOCIOController.php (73)
in C:\xampp\htdocs\FACIS\index.php (13)
2013/10/22 23:12:01 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1400 OCIStmtExecute: ORA-01400: cannot insert NULL into ("FACIS"."SOCIO"."K_IDENTIFICACION")
 (ext\pdo_oci\oci_statement.c:148). The SQL statement executed was: INSERT INTO "SOCIO" ("N_NOMBRE", "N_APELLIDO", "I_ESTADO_CIVIL", "N_OCUPACION", "O_TARJETA_PROFESIONAL", "I_GENERO", "O_DIRECCION_DOMICILIO", "O_DIRECCION_TRABAJO", "O_CORREO_ELECTRONICO", "O_TELEFONO_DOMICILIO", "O_TELEFONO_TRABAJO", "O_TELEFONO_CELULAR", "F_INGRESO", "F_RETIRO", "O_CAUSAL_RETIRO") VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14) RETURNING "K_IDENTIFICACION" INTO :RETURN_ID' in C:\xampp\htdocs\FACIS\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\xampp\htdocs\FACIS\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\xampp\htdocs\FACIS\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\FACIS\protected\controllers\SOCIOController.php(73): CActiveRecord->save()
#3 C:\xampp\htdocs\FACIS\framework\web\actions\CInlineAction.php(49): SOCIOController->actionCreate()
#4 C:\xampp\htdocs\FACIS\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\FACIS\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\FACIS\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\FACIS\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\FACIS\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\FACIS\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\FACIS\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\FACIS\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\FACIS\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\htdocs\FACIS\framework\web\CWebApplication.php(141): CWebApplication->runController('sOCIO/create')
#14 C:\xampp\htdocs\FACIS\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\FACIS\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/facis/index.php/sOCIO/create
HTTP_REFERER=http://localhost:8081/facis/index.php/sOCIO/create
---
2013/10/22 23:13:02 [warning] [application] Failed to set unsafe attribute "K_IDENTIFICACION" of "SOCIO".
in C:\xampp\htdocs\FACIS\protected\controllers\SOCIOController.php (72)
in C:\xampp\htdocs\FACIS\index.php (13)
2013/10/22 23:13:02 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1861 OCIStmtExecute: ORA-01861: literal does not match format string
 (ext\pdo_oci\oci_statement.c:148). The SQL statement executed was: INSERT INTO "SOCIO" ("N_NOMBRE", "N_APELLIDO", "I_ESTADO_CIVIL", "N_OCUPACION", "O_TARJETA_PROFESIONAL", "I_GENERO", "O_DIRECCION_DOMICILIO", "O_DIRECCION_TRABAJO", "O_CORREO_ELECTRONICO", "O_TELEFONO_DOMICILIO", "O_TELEFONO_TRABAJO", "O_TELEFONO_CELULAR", "F_INGRESO", "F_RETIRO", "O_CAUSAL_RETIRO", "K_IDENTIFICACION") VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14, :yp15) RETURNING "K_IDENTIFICACION" INTO :RETURN_ID.
in C:\xampp\htdocs\FACIS\protected\controllers\SOCIOController.php (74)
in C:\xampp\htdocs\FACIS\index.php (13)
2013/10/22 23:13:02 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1861 OCIStmtExecute: ORA-01861: literal does not match format string
 (ext\pdo_oci\oci_statement.c:148). The SQL statement executed was: INSERT INTO "SOCIO" ("N_NOMBRE", "N_APELLIDO", "I_ESTADO_CIVIL", "N_OCUPACION", "O_TARJETA_PROFESIONAL", "I_GENERO", "O_DIRECCION_DOMICILIO", "O_DIRECCION_TRABAJO", "O_CORREO_ELECTRONICO", "O_TELEFONO_DOMICILIO", "O_TELEFONO_TRABAJO", "O_TELEFONO_CELULAR", "F_INGRESO", "F_RETIRO", "O_CAUSAL_RETIRO", "K_IDENTIFICACION") VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14, :yp15) RETURNING "K_IDENTIFICACION" INTO :RETURN_ID' in C:\xampp\htdocs\FACIS\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\xampp\htdocs\FACIS\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\xampp\htdocs\FACIS\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\FACIS\protected\controllers\SOCIOController.php(74): CActiveRecord->save(false)
#3 C:\xampp\htdocs\FACIS\framework\web\actions\CInlineAction.php(49): SOCIOController->actionCreate()
#4 C:\xampp\htdocs\FACIS\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\FACIS\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\FACIS\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\FACIS\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\FACIS\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\FACIS\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\FACIS\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\FACIS\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\FACIS\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\htdocs\FACIS\framework\web\CWebApplication.php(141): CWebApplication->runController('sOCIO/create')
#14 C:\xampp\htdocs\FACIS\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\FACIS\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/facis/index.php/sOCIO/create
HTTP_REFERER=http://localhost:8081/facis/index.php/sOCIO/create
---
2013/10/22 23:15:43 [warning] [application] Failed to set unsafe attribute "K_IDENTIFICACION" of "SOCIO".
in C:\xampp\htdocs\FACIS\protected\controllers\SOCIOController.php (72)
in C:\xampp\htdocs\FACIS\index.php (13)
